CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to this project will be documented in this file.
 
 ## Changed
 
+* Remove wrapper `openssl::_EVP_PKEY_CTX_get_rsa_oaep_label`
```

Comment:
As discussed, let's document here why we're not bumping the SOVERSION for this, even though this is technically an ABI break.

Reply:
Done.